src/components/LoadingScreen/LoadingScreen.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./LoadingScreen.css";
+
+const LoadingScreen = props => {
```

Comment:
...or, you know, https://www.npmjs.com/package/react-loading-screen / https://github.com/tanem/react-nprogress. 😉